src/platform/patmos/patmos.c

```diff
@@ -20,7 +21,7 @@ instant_t PlatformPatmos_get_physical_time(Platform *super) {
 lf_ret_t PlatformPatmos_wait_until_interruptible(Platform *super, instant_t wakeup_time) {
   PlatformPatmos *self = (PlatformPatmos *)super;
   self->async_event = false;
-  super->leave_critical_section(super); // turing on interrupts
+  //super->leave_critical_section(super); // turing on interrupts
```

Comment:
why not remove it? we moved away from those critical sections anyway, and instead now have a mutex abstraction.